### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by AllenM97** [MCG-NJU/TDN#13]
Hi, your paper says "...the initial learning rate is 0.02... The learning rate will be devided by a factor of 10 when the performance on validation **set saturates**." And what argument should I put in training so that to achieve this kind of decay of learning rate? Thank you very much!

**Comment by yztongzhan** [comment]
Hi @AllenM97 ! We use normal `MultiStepLR` in PyTorch to adjust the learning rate.
